### ISSUE TITLE
Update pip repository

### ISSUE THREAD

**Issue by omarryhan** [plamere/spotipy#311]
Hey,
Can you please update the pip repository as it's lagging behind Github's code?
Thanks

**Comment by txoof** [comment]
The util.py is out of sync with the git repo. Would you please update the pip repository?
PIP:
```
def prompt_for_user_token(username, scope=None, client_id = None,
        client_secret = None, redirect_uri = None):
    ''' prompts the user to login if necessary and returns
        the user token suitable for use with the spotipy.Spotify
        constructor

        Parameters:

         - username - the Spotify username
         - scope - the desired scope of the request
         - client_id - the client id of your app
         - client_secret - the client secret of your app
         - redirect_uri - the redirect URI of your app
```
Git Repo:
```
def prompt_for_user_token(username, scope=None, client_id = None,
        client_secret = None, redirect_uri = None, cache_path = None):
    ''' prompts the user to login if necessary and returns
        the user token suitable for use with the spotipy.Spotify 
        constructor
        Parameters:
         - username - the Spotify username
         - scope - the desired scope of the request
         - client_id - the client id of your app
         - client_secret - the client secret of your app
         - redirect_uri - the redirect URI of your app
         - cache_path - path to location to save tokens
```

**Comment by stephanebruckert** [comment]
This should be fixed with version 2.5.0, just do:

    pip install spotipy --upgrade
